src/components/Editor.tsx

```diff
@@ -510,7 +510,7 @@ export const Editor = ({
   );
 
   useReceiveMessage("getThemeData", TARGET_ORIGINS, (send, payload) => {
-    console.log(`payload in getThemeData is ${JSON.stringify(payload)}`);
+    devLogger.log("getThemeData PAYLOAD: " + JSON.stringify(payload));
```

Comment:
In the other useReceiveMessages we have ie `devLogger.logData("SAVE_STATE", payload);`, can you do the same? And then remove the devlogger below that logs it again after it's json escaped. We don't really need it twice. Same in the others.